x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/detector_description.tsx

```diff
@@ -63,49 +58,82 @@ function createDetectorDescription(jobCreator: RareJobCreator, detectorType: RAR
-      defaultMessage: 'detects rare values of {rareFieldName}',
+      defaultMessage: 'Detects rare {rareFieldName} values.',
       values: { rareFieldName },
     }
   );
 
-  const beginningSummaryFreq = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummaryFreq',
+  const rareSplitSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSplitSummary',
     {
-      defaultMessage: 'detects frequently rare values of {rareFieldName}',
-      values: { rareFieldName },
+      defaultMessage: 'For each {splitFieldName}, detects rare {rareFieldName} values.',
+      values: { splitFieldName, rareFieldName },
     }
   );
 
-  const population = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.population',
+  const freqRarePopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.freqRarePopulationSummary',
     {
-      defaultMessage: 'compared to the population of {populationFieldName}',
-      values: { populationFieldName },
+      defaultMessage:
+        'Detects {populationFieldName} values that frequently have rare {rareFieldName} values relative to the population.',
+      values: { populationFieldName, rareFieldName },
     }
   );
 
-  const split = i18n.translate('xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.split', {
-    defaultMessage: 'for each value of {splitFieldName}',
-    values: { splitFieldName },
-  });
+  const freqRareSplitPopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.freqRareSplitPopulationSummary',
+    {
+      defaultMessage:
+        'For each {splitFieldName}, detects {populationFieldName} values that frequently have rare {rareFieldName} values relative to the population.',
+      values: { splitFieldName, populationFieldName, rareFieldName },
+    }
+  );
+
+  const rarePopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rarePopulationSummary',
+    {
+      defaultMessage:
+        'Detects {populationFieldName} values that have rare {rareFieldName} values relative to the population.',
+      values: { populationFieldName, rareFieldName },
+    }
+  );
+
+  const rareSplitPopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSplitPopulationSummary',
+    {
+      defaultMessage:
+        'For each {splitFieldName}, detects {populationFieldName} values that have rare {rareFieldName} values relative to the population.',
+      values: { splitFieldName, populationFieldName, rareFieldName },
+    }
+  );
 
   const desc = [];
 
-  if (detectorType === RARE_DETECTOR_TYPE.FREQ_RARE_POPULATION) {
-    desc.push(beginningSummaryFreq);
-  } else {
-    desc.push(beginningSummary);
+  if (detectorType === RARE_DETECTOR_TYPE.RARE) {
+    if (splitFieldName !== undefined) {
+      desc.push(rareSplitSummary);
```

Comment:
Seeing as we only display a single bullet point now, I think `description` can now be a `string` rather than `string[]` so the values can simply be returned rather than adding to an array.

Reply:
This change (and your other code cleanup suggestions) are in 080289cc4307f3ec4cf505ff217c385467971510